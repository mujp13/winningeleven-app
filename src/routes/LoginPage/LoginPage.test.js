import React from "react";
import ReactDOM from "react-dom";
import LoginPage from "./LoginPage";
import { MemoryRouter } from "react-router-dom";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(
    <MemoryRouter>
      <LoginPage />
    </MemoryRouter>,
    div
  );

  ReactDOM.unmountComponentAtNode(div);
});

it("renders correctly", () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <LoginPage />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
