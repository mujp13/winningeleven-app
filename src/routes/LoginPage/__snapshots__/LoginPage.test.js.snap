// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="Section LoginPage"
>
  <h2>
    Login
  </h2>
  <form
    className="LoginForm"
    onSubmit={[Function]}
  >
    <div
      role="alert"
    />
    <div
      className="user_name"
    >
      <label
        htmlFor="LoginForm__user_name"
      >
        User name
      </label>
      <input
        className="Input "
        id="LoginForm__user_name"
        name="user_name"
        required={true}
      />
    </div>
    <div
      className="password"
    >
      <label
        htmlFor="LoginForm__password"
      >
        Password
      </label>
      <input
        className="Input "
        id="LoginForm__password"
        name="password"
        required={true}
        type="password"
      />
    </div>
    <button
      className="Button "
      type="submit"
    >
      Login
    </button>
  </form>
</section>
`;
